src/main/java/nextstep/security/role/RoleHierarchyImpl.java

```diff
@@ -0,0 +1,95 @@
+package nextstep.security.role;
+
+import org.springframework.util.StringUtils;
+
+import java.util.*;
+
+public class RoleHierarchyImpl implements RoleHierarchy {
+
+    Map<String, Set<GrantedAuthority>> roleHierarchy;
+
+    private static final List<GrantedAuthority> EMPTY_AUTHORITIES = Collections.emptyList();
```

Comment:
```suggestion
    private static final List<GrantedAuthority> EMPTY_AUTHORITIES = Collections.emptyList();

    private final Map<String, Set<GrantedAuthority>> roleHierarchy;
```

 상수는 자바 컨벤션상 가장 위에 있어야하고, `roleHierarchy`는 접근제어가 필요해보이네요 :)

Reply:
수정하겠습니다!